src/pages/management/configuration-api/v3.5/index.mdx

```diff
@@ -2909,6 +2909,65 @@ curl -X GET \
 
 </Section>
 
+<Section>
+
+<Text>
+
+### Get Product Limits for Plan
+
+LiveChat Platform limits some resources to keep infrastructure healthy and stable for all clients. The limits are different for each pricing plan. The method `check_products_limits_for_plan` shows which resources are exceeding for any given plan.
+
+Limited resources in LiveChat:
+
+|                        |                                                                                                                     |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------- |
+| `agents`               | The max number of agents can be invited                                                                             |
```

Comment:
This is about active agents (including bots, excluding suspended or awaiting approval)

Reply:
good point, added as a note https://github.com/livechat/livechat-public-docs/pull/1099/commits/294df83936678ed94edbcdb382631e5b1a9c26c5